src/main/java/Deadline.java

```diff
@@ -0,0 +1,32 @@
+import java.time.LocalDateTime;
+
+public class Deadline extends Task {

```

Comment:
Perhaps can put this class into a package (like package duke), as other developers could better understand the code base when all the classes have been grouped in packages